src/wp-includes/block-bindings/sources/pattern.php

```diff
@@ -16,6 +16,8 @@ function pattern_source_callback( $source_attrs, $block_instance, $attribute_nam
 
 wp_block_bindings_register_source(
 	'pattern_attributes',
-	__( 'Pattern Attributes' ),
-	'pattern_source_callback'
+	array(
+		__( 'Pattern Attributes' ),
```

Comment:
Keys are missing for both properties.

Reply:
And that's precisely why I wanted to change the signature in the first place 🙃 .

Thanks for catching this.

Reply:
updated in 9f2274d354b42ebed7d510edee596d9f1085d22d